index.html

```diff
@@ -201,10 +201,10 @@ <h4>CANOPY</h4>
 
              </div>
 
-         </div>
+         </main>
 
 
-         <div class="trabajo foto-izquierda tamaño flex-column">
+         <main class="trabajo foto-izquierda tamaño flex-column">
```

Comment:
- I suggest you change the <main> tag to <section> as it is recommended by good practice for semantic structure, accessibility, styling, organization and maintainability of the code.